doc/stdlib/arrays.rst

```diff
@@ -187,7 +187,7 @@ Constructors
 
    Create an array of all zeros with the same element type and shape as ``A``\ .
 
-.. function:: ones([T::Type=Float64,] dims)
+.. function:: ones(type, dims)
```

Comment:
No, you have it backwards. We have two docstrings for one method if you use the empty signatures.